test/test_models.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import random
 from pyronear import models
+import torchvision
```

Comment:
Codacy seems to say that this is an unused import, you can remove it ;)

Reply:
ok thanks